stable/redis-ha/templates/_configs.tpl

```diff
@@ -163,7 +163,7 @@
       {{- end }}
       tcp-check send PING\r\n
       tcp-check expect string +PONG
-      tcp-check send SENTINEL\ get-master-addr-by-name\ mymaster\r\n
+      tcp-check send SENTINEL\ get-master-addr-by-name\ {{ .Values.redis.masterGroupName }}\r\n
```

Comment:
==> Linting stable/redis-ha
[ERROR] templates/: render error in "redis-ha/templates/redis-haproxy-deployment.yaml": template: redis-ha/templates/redis-haproxy-deployment.yaml:27:35: executing "redis-ha/templates/redis-haproxy-deployment.yaml" at <include "config-hapr...>: error calling include: template: redis-ha/templates/_configs.tpl:166:66: executing "config-haproxy.cfg" at <.Values.redis.master...>: can't evaluate field Values in type int

Reply:
Got it fixed, thanks